### PR TITLE
sustech-bachelor-thesis:0.1.0

### PR DESCRIPTION
**Pull request by Duolei-Wang**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: 
The typst template for SUSTech bachelor thesis.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/sustech-bachelor-thesis/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "sustech-bachelor-thesis"
+version = "0.1.0"
+entrypoint = "lib.typ"
```
Comment (istudyatuni) [inline packages/preview/sustech-bachelor-thesis/0.1.0/typst.toml:4]: Probably you want 

```suggestion
entrypoint = "sections/lib.typ"
```

Comment (Duolei-Wang) [reply packages/preview/sustech-bachelor-thesis/0.1.0/typst.toml]: Really thanks a lot! I'm really confused by the meaning of 'entrypoint'. But after I rebuild my template locally, I rebuild my template in a directly way. I will construct them in the later versions.

Comment (Duolei-Wang) [reply packages/preview/sustech-bachelor-thesis/0.1.0/typst.toml]: Really thanks a lot! I'm really confused by the meaning of 'entrypoint'. But after I build my template locally, I move my main files in the folder sections. I will construct them better in the later versions.

**Comment by Duolei-Wang** [comment]
A first try of my self-used version. Though I was struggling with template upload.
I will try to make the template better. qwq

**Comment by Duolei-Wang** [comment]
And the version 0.1.0 make all the configuration files available in templates, which intends users can create their own versions as it's really a framework. 

The later releases will remain only content.typ, appendix.typ and reconstruct the style control. When it's stable enough and need no more style changes, it will be marked as version0.2.0. 

Any suggestions please contact me! Goodnight!

**Pull request closed without merging**